src/utils.jl

```diff
@@ -179,11 +179,13 @@ ispunctuation(t::Token) = t.kind == Tokens.COMMA ||
 
 isstring(x) = false
 isstring(x::EXPR{StringH}) = true
-isstring(x::LITERAL{Tokens.STRING}) = true
-isstring(x::LITERAL{Tokens.TRIPLE_STRING}) = true
+isstring(x::LITERAL) = x.kind == Tokens.STRING || x.kind == Tokens.TRIPLE_STRING
+is_integer(x) = x isa LITERAL && x.kind == Tokens.INTEGER
+is_float(x) = x isa LITERAL && x.kind == Tokens.FLOAT
+is_number(x) = x isa LITERAL && (x.kind == Tokens.INTEGER || x.kind == Tokens.FLOAT)
+is_nothing(x) = x isa LITERAL && x.kind == Tokens.begin_plus
```

Comment:
Huh?

Reply:
I need to fix this, I use a couple barrier tokens as Token kinds in CSTParser. I'll add a PR which does so and removes the `invisible_keyword`s from Tokenize